scripts/pack.ps1

```diff
@@ -1,6 +1,9 @@
 New-Item "package-release" -ItemType Directory
 
 # Copy `package-dev` stuff
+dotnet tool install --global Alias --version 0.1.0
```

Comment:
We'll need to have this run through msbuild so that when we run `pack.ps1`, assemblies are already aliased.
The reason is that when we load up Unity with the sample project, we'd things already aliased, so unit and integration tests will be loaded aliased. And when we build players for testing or smoke testing, they'll be validating the bits we'll bundle through this script

Reply:
@bruno-garcia in the after build of the test project?

Reply:
`install` skips even if the provided differs from the already installed one?